package/com.unity.formats.usd/Tests/Common/TestUtilityFunction.cs

```diff
@@ -0,0 +1,34 @@
+using NUnit.Framework;
+using pxr;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+using USD.NET.Unity;
+
+public static class TestUtilityFunction
+{
+#if UNITY_EDITOR
+    public static class EditorTest
+    {
+        public static Scene OpenUSDScene(string usdGUID, UsdStage.InitialLoadSet loadSet = UsdStage.InitialLoadSet.LoadNone)
+        {
+            var usdPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(usdGUID));
+            var stage = UsdStage.Open(usdPath, loadSet);
+            var scene = Scene.Open(stage);
+
+            return scene;
+        }
+    }
+#endif
+    public static class ExportTest
```

Comment:
This isn't export specific, so I think this extra layer of abstraction will make it hard to find this for reuse. I think both of these functions can just be inside 'TestUtilityFunctions'